stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/IdentifierSpec.kt

```diff
@@ -5,6 +5,11 @@ import androidx.annotation.RestrictTo
 import kotlinx.parcelize.Parcelize
 import kotlinx.serialization.Serializable
 
+enum class RequestDestination {
```

Comment:
I know this is my name suggestion, but maybe there’s something more descriptive 🙈

Reply:
how about `ApiParameterDestination`?

Reply:
I like that!